docs/pages/modules/get-started.mdx

```diff
@@ -1,118 +1,42 @@
-```
-
-</CodeBlocksTable>
-
-### Add Expo packages to dependencies
-
-Add `expo` package as a peer dependency in your **package.json** — we recommend using `*` as a version range so as not to cause any duplicated packages in user's **node_modules** folder. Your library also needs to depend on `expo-modules-core` but only as a dev dependency — it's already provided in the projects depending on your library by the `expo` package with the version of core that is compatible with the specific SDK used in the project.<br/>
-
-```json package.json
-{
-  // ...
-  "devDependencies": {
-    "expo-modules-core": "^X.Y.Z"
-  },
-  "peerDependencies": {
-    "expo": "*"
-  }
-}
-```
-
-## Create a native module
-
-To use the Expo Modules API for native modules, you need to [set up your library as an Expo module](#set-up-a-library-as-an-expo-module). Once complete, create Swift and Kotlin files from the templates below.
+To create a new Expo module from scratch, run the `create-expo-module` script as shown below. The script will ask you a few questions and then generate the native Expo module along with the example app for iOS and Android that uses your new module.
 
-<CodeBlocksTable tabs={["MyModule.swift", "MyModule.kt"]}>
+<Terminal cmd={[`$ npx create-expo-module my-module`]} />
 
-```swift
-import ExpoModulesCore
+## 2. Running the example project
 
-public class MyModule: Module {
-  public func definition() -> ModuleDefinition {
-    // Definition components go here
-  }
-}
-```
+> **Note:** If you're using Windows, you can open the example project by opening the `android` and/or `ios` directory in Android Studio and Xcode respectively,
 
-```kotlin
-package my.module.package
+Navigate to the module directory and then open the iOS and/or Android example project by running the following commands:
 
-import expo.modules.kotlin.modules.Module
-import expo.modules.kotlin.modules.ModuleDefinition
+<Terminal cmd={[`$ cd my-module`, `$ npm run open:android`, `$ npm run open:ios`]} />
 
-class MyModule : Module() {
-  override fun definition() = ModuleDefinition {
-    // Definition components go here
-  }
-}
-```
+Now, you can run the example project on your device or simulator/emulator. When the project compiles and runs, you will see "Hello world! 👋" on the screen.
 
-</CodeBlocksTable>
+## 3. Making a change
 
-Then, add your classes to Android and/or iOS `modules` in the [module config](./module-config.mdx). Expo Autolinking will automatically link these classes as native modules in the user's project.
+### iOS
 
-```json expo-module.config.json
-{
-  "ios": {
-    "modules": ["MyModule"]
-  },
-  "android": {
-    "modules": ["my.module.package.MyModule"]
-  }
-}
-```
+Open up `MyModuleModule.swift` in Xcode (<kbd>⌘+o</kbd> or <kbd>ctrl+o</kbd> and search for `MyModuleModule.swift`) and change the `hello` method to return a different string. For example, you can change it to return `"Hello world! 🌎🍎"`. Re-build the app and you should see your change.
```

Comment:
```suggestion
Open up `MyModuleModule.swift` in Xcode (<kbd>⌘</kbd> + <kbd>o</kbd> or <kbd>Ctrl</kbd> + <kbd>o</kbd> and search for `MyModuleModule.swift`) and change the `hello` method to return a different string. For example, you can change it to return `"Hello world! 🌎🍎"`. Re-build the app and you should see your change.
```

Corrected keyboard shortcuts.

Reply:
Adding one change to Simek's suggestion: isn't `rebuild` more common than `re-build`?
```suggestion
Open up `MyModuleModule.swift` in Xcode (<kbd>⌘</kbd> + <kbd>O</kbd> or <kbd>Ctrl</kbd> + <kbd>O</kbd> and search for `MyModuleModule.swift`) and change the `hello` method to return a different string. For example, you can change it to return `"Hello world! 🌎🍎"`. Rebuild the app and you should see your change.
```